packages/react-union/src/components/Union/Union.js

```diff
@@ -74,15 +74,14 @@ class Union extends Component {
 			},
 			error => onScanError(error)
 		);
-	}
+	};
+
+	renderWidget = config => (
+		<Widget key={config.descriptor.namespace || config.descriptor.container} {...config} />
+	);
 
 	render() {
-		return (
-			<div>
-				{this.props.children}
-				<Widgets configs={this.state.configs} />
-			</div>
-		);
+		return [this.props.children, this.state.configs.map(this.renderWidget)];
```

Comment:
Return one div element not an array. I wanna Union to be compatible with component providers (e.g.  react-redux's Provider component or IntlProvider, ...) that are using `Children.only` validation.

Reply:
Good call. At first, I wanted to use React.Fragment but I didn't realise it's supported from React v16.2, so I replaced it with an array.